parts/kubernetesmaster-kube-addon-manager.yaml

```diff
@@ -3,7 +3,6 @@ kind: Pod
 metadata:
   name: kube-addon-manager
   namespace: kube-system
-  version: v1
```

Comment:
this is an unnecessary `version` metadata tag for `kube-addon-manager`, good change, let's put this in a separate PR